core/commanddefinition.js

```diff
@@ -160,3 +160,46 @@
  * @since 4.6.0
  * @property {Number} fakeKeystroke
  */
+
+/**
+ * Provides command's label string. Label is define with few fallbacks in specific order described below:
+ *
+ * 1. If {@link CKEDITOR.command#getLabel} method exists, it is used to provide label,
+ * 2. If {@link CKEDITOR.commandDefinition#label} atribute exists, it is used to provide label.
+ * 3. If {@link CKEDITOR.command#uiItems} is non empty, then label of first array's item is used as label
+ * 4. Empty string.
+ *
+ * @since 4.8.0
+ * @method getLabel
+ * @param {CKEDITOR.editor} editor The editor instance.
+ * @returns {String} command's label
+ */
+
+/**
+ * Provides command's description. Description is define with few fallbacks in specific order described below:
+ *
+ * 1. If {@link CKEDITOR.command#getKeyDescription} method exists, it is used to provide description,
+ * 2. If {@link CKEDITOR.commandDefinition#keyDescription} atribute exists, it is used to provide description.
+ * 3. Empty string
+ *
+ * @since 4.8.0
+ * @method getKeyDescription
+ * @param {CKEDITOR.editor} editor The editor instance.
+ * @returns {String} command's description
+ */
+
+/**
+ * Defines command label. This property is used by different plugins to display command information (by e.g. Accessibility Help).
+ * For geting command's label please use {@link CKEDITOR.commandDefinition#getLabel}, instead of directly reading `label` property.
+ *
+ * @since 4.8.0
```

Comment:
I have doubts about it. If those will be marked as `@private`, then some kind of 'setter' might be required. Users might not see those variables, or be afraid of modifying them. What do you think about that?